client/src/settings/app.tsx

```diff
@@ -228,9 +228,7 @@ function Settings({
       )}
 
       <div className="field-group">
-        <label htmlFor="id_locale" className="slab-highlight">
-          Change language
-        </label>
```

Comment:
All our form elements need to have an associated label for accessibility. To address the visual aspect here I would do the following:

1. remove the class from the `label` element.
2. Then replace this:

```css
.field-group {
  margin: math.div($base-spacing, 2) 0;

  label {
    font-size: $large-font-size-mobile;
    line-height: $heading-line-height;
    margin-bottom: $base-spacing;

    @media #{$mq-tablet-and-up} {
      font-size: $large-font-size;
    }
  }
}
```

in `client/src/settings/index.scss` with:

```
@import "~@mdn/minimalist/sass/mixins/typography";

.field-group {
  margin: math.div($base-spacing, 2) 0;

  label {
    @include headings-common-base();
    @include heading-3();
    margin-bottom: $base-spacing;
  }
}
```

Reply:
Wonderful! But... before I jump into that. How about we make the "Close account" text a label for the button below?
That way both sides use `<label>` instead of `<h3>`. 

It's just a bit weird that we're abusing labels as headings. 
Perhaps it ought to be that both boxes have a `<h3>` that is easy to understand. Then, for the `<select>` we have a dedicated `<label>`. 

Perhaps:
```html
<h3>Default language</h3>

<label>Change language</label>
<select>...</select>
```

Reply:
Yes, I prefer the second option aka:

```
<h3>Default language</h3>

<label>Change language</label>
<select>...</select>
```

Reply:
Feels more semantic and just "right". Now the title is a title instead of a verb.

Reply:
Done. But now it looks like this:
<img width="862" alt="Screen Shot 2021-05-25 at 2 34 11 PM" src="https://user-images.githubusercontent.com/26739/119550382-5c6ea100-bd66-11eb-9982-8ba34513d9bb.png">

Would it be an option to make the `<label>` invisible for people not using a screenreader.

Reply:
Ah yes, with the change you made, we can just remove this entire piece of SASS:

```css
label {
    font-size: $large-font-size-mobile;
    line-height: $heading-line-height;
    margin-bottom: $base-spacing;

    @media #{$mq-tablet-and-up} {
      font-size: $large-font-size;
    }
  }
```